Training/2.0 Libraries.md

```diff
@@ -25,14 +25,20 @@ Make a two-dimensional numpy array. First row will be: [1,2,3] and second row wi
 Using numpy functionality find the mean, minimum and max value of your array. Index the maximum value and assign it to a variable named max. Index the minimum value and assign it to a variable named min. 
 
 ```
-
+x = numpy.array([1,2,3],[4,5,6])
+max = x.max()
+min = x.min()
+mean = x.mean()
 ```
 
```

Comment:
To correctly make the array, you have to put another set of square brackets. Also, you have already imported numpy as np, so you can use this when creating the array
A better option for the first line would be:    x = np.array([[1,2,3],[4,5,6]])